pkg/zpdb/cache.go

```diff
@@ -0,0 +1,93 @@
+package zpdb
+
+import (
+	"errors"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type Cache struct {
+	// name --> config
+	cache map[string]*Config
```

Comment:
[nit] Given the struct is called "cache", perhaps something like "entries" would be a less confusing name here?